pkg/policy/report.go

```diff
@@ -213,6 +217,41 @@ func eraseResultEntries(pclient kyvernoclient.Interface, reportLister policyrepo
 	return fmt.Errorf("failed to erase results entries %v", strings.Join(errors, ";"))
 }
 
+func eraseSplitResultEntries(pclient kyvernoclient.Interface, ns *string, selector labels.Selector) error {
+	var errors []string
+
+	if ns != nil {
```

Comment:
We still need a NS check to decide whether to erase polr or cpolr:
```
if ns != nil{
  if *ns != ""{
    // erase polrs
  } else{
    // erase cpolrs
  }
}
```

Reply:
hmm, ok done